tests/test_project_manager.py

```diff
@@ -932,9 +994,42 @@ def test_total_capex():
     fix_project = ProjectManager(complete_project)
     fix_project.run()
 
-    assert fix_project.total_capex == pytest.approx(1207278397.56, abs=1e-1)
+    assert fix_project.total_capex == pytest.approx(
+        1216449001.7410436, abs=1e-1
+    )
 
     flt_project = ProjectManager(complete_floating_project)
     flt_project.run()
 
-    assert flt_project.total_capex == pytest.approx(3284781912.73, abs=1e-1)
+    assert flt_project.total_capex == pytest.approx(
+        3540761314.148985, abs=1e-1
+    )
+
+
+def test_construction_financing_factor_exception():
+    """Test the construction financing factor method for soft capex."""
+
+    config = deepcopy(complete_project)
+    config["project_parameters"] = {
+        "spend_schedule": {0: 0.15, 1: 0.25, 2: 0.3, 3: 0.1, 4: 0.1, 5: 0.0}
+    }  # does not sum to 1.0
+
+    with pytest.raises(ValueError):
+
+        project = ProjectManager(config)
+        project.construction_financing_factor()
+
+
+def test_deprecated_warnings():
+
+    config = deepcopy(complete_project)
+    config["project_parameters"] = {"commissioning": 44}
```

Comment:
Need to delete this test if we are going with commissioning, but leave the check for contingency. The test will fail as is

Reply:
Good catch, just updated that. Thanks.